image/v3/modifying/drawing.md

```diff
@@ -219,11 +219,11 @@ $image = $manager->read('test.png');
 
 // draw a polygon
 $image->drawPolygon(function ($polygon) {
-    $polygon->point(10, 10); // add point of polygon
-    $polygon->point(150, 150); // add point
-    $polygon->point(40, 180); // add point
-    $polygon->point(60, 100); // add point
-    $polygon->background('#b35187'); // background color
-    $polygon->border('#ff0', 6); // border color and border width
+    $polygon->addPoint(10, 10); // add point of polygon
+    $polygon->addPoint(150, 150); // add point
+    $polygon->addPoint(40, 180); // add point
+    $polygon->addPoint(60, 100); // add point
+    $polygon->setBackgroundColor('#b35187'); // background color
+    $polygon->setBorder('#ff0', 6); // border color and border width
```

Comment:
Please revert. These were correct calls of class `Intervention\Image\Geometry\Factories\PolygonFactory::class`.